release-notes/7.0/known-issues.md

```diff
@@ -100,115 +100,29 @@ More information and workaround can be found at <https://github.com/dotnet/runti
 
 ## .NET SDK
 
-### [7.0] Projects using certain workloads don't load, build, and or run if .NET 7 Preview SDK workloads are installed
+### Using Visual Studio 17.6 to target .NET 6 may hit NETSDK1145 if nuget.org not configured for package downloads
 
-If a preview .NET 7 SDK is installed, projects with workload dependencies such as `microsoft.net.workload.mono.toolchain` may fail to build, load, and or run. An example of this issue is described [here](https://github.com/dotnet/sdk/issues/28947).
+.NET 7 went [out of support](https://devblogs.microsoft.com/dotnet/dotnet-7-end-of-support/) in May 2024. Per the plan outlined in that blog post, we have included the .NET 7 SDK in 17.6 as optional but are no longer updating it. We will continue to update the 6.0 runtime included in Visual Studio 17.6 to ensure customers have the latest security fixes when targeting .NET 6. However, when building using the .NET 7 SDK, it will expect Host Packs, Targeting Packs, and Runtime Packs from an older version of .NET 6 than is installed by default. It will attempt to download these packages from NuGet.org.
```

Comment:
```suggestion
.NET 7 went [out of support](https://devblogs.microsoft.com/dotnet/dotnet-7-end-of-support/) in May 2024. Per the plan outlined in that blog post, we have included the .NET 7 SDK in 17.6 as an optional component but are no longer updating it. We will continue to update the 6.0 runtime included in Visual Studio 17.6 to ensure customers have the latest security fixes when targeting .NET 6. However, when building using the .NET 7 SDK, it will expect Host Packs, Targeting Packs, and Runtime Packs from an older version of .NET 6 than is installed by default. It will attempt to download these packages from NuGet.org.
```